swagger-ui-react: Module not found: Can't resolve '@babel/runtime-corejs2'...

### Q&A
 - Method of installation: yarn add swagger-ui-react
 - Swagger-UI version: 3.23.2

### Content & configuration

Clean create-react-app project

### Describe the bug you're encountering

Cannot compile because of the error with `@babel/runtime-corejs2` missing:

```
Failed to compile.

./node_modules/swagger-ui-react/index.js
Module not found: Can't resolve '@babel/runtime-corejs2/core-js/json/stringify' in 'C:\Users\username\project\node_modules\swagger-ui-react'
```

### To reproduce...

Steps to reproduce the behavior:
1. Create a new react project using create-react-app
2. `yarn add swagger-ui-react`
3. `import SwaggerUI from "swagger-ui-react";`
4. `yarn start`

### Expected behavior

It can compile with `swagger-ui-react` 3.23.0

### Additional context or thoughts

`@babel/runtime-corejs2` apparently is not used in create-react-app projects. So it should be declared as `dependencies` for `swagger-ui-react`. Adding it explicitly to `package.json` is not a common practice.

--- a/package.json
+++ b/package.json
@@ -47,6 +47,7 @@
     "start": "npm-run-all --parallel serve-static open-static"
   },
   "dependencies": {
+    "@babel/runtime-corejs2": "^7.5.5",
     "@braintree/sanitize-url": "^2.0.2",
     "@kyleshockey/js-yaml": "^1.0.1",
     "@kyleshockey/object-assign-deep": "^0.4.2",
@@ -91,7 +92,6 @@
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.4.4",
-    "@babel/runtime-corejs2": "^7.0.0",
     "@release-it/conventional-changelog": "^1.1.0",
     "autoprefixer": "^8.4.1",
     "babel-eslint": "^9.0.0",

